### ISSUE TITLE
Choosing an installation directory for kurl

### ISSUE THREAD

**Issue by namgk** [replicatedhq/kURL#4450]
Hi there,

We are using kurl in Azure's VM. Azure's OS disk is only 30GB so when kurl run, it bailed with "the disk containing directory /var/lib/kubelet" has less than 30GB".

We'd have to mount a separate data disk to a location, such as /mnt/datadisk, but how do we point kurl to that new location so it can install?

Thanks

**Comment by rrpolanco** [comment]
@namgk yes, the docker images and other assets are stored at the `kurl-install-directory` path.
